docs/configuration/config-file-reference.md

```diff
@@ -126,6 +126,11 @@ api:
 # Cortex query-frontend.
 [query_range: <query_range_config>]
 
+query:
+  # Number of shards to use when distributing shardable PromQL queries.
+  # CLI flag: -querier.vertical-shard-size
+  [vertical_shard_size: <int> | default = 0]
+
```

Comment:
I think for now we should se this as experimental.

Reply:
+1

Reply:
Ok...

I added as per tenant configuration (runtimeConfig) and hide it from docs.

Reply:
Now we can enable the feature by tenant.

@alvinlin123 make sense?